src/main/kotlin/org/kamiblue/client/gui/rgui/component/Slider.kt

```diff
@@ -114,7 +116,9 @@ open class Slider(
                     (renderHeight * ClickGUI.getScaleFactor()).roundToInt()
             )
         }*/
-        FontRenderAdapter.drawString(name, 1.5f, 1.0f, color = GuiColors.text)
+        val text = if (isBold()) TextFormatting.BOLD.format(name) else name
```

Comment:
You don't need `TextFormatting.BOLD.format(name)`, you can just do `TextFormatting.BOLD format name`

Reply:
Will correct this tomorrow.